src/transforms/samplesTransforms.ts

```diff
@@ -1,349 +1,361 @@
+          }
+          if (clientParameterNames.length > 0) {
+            sample.clientParameterNames = clientParameterNames.join(", ");
+          }
+          const methodParameterNames = [];
+          const optionalParams: [string, string][] = [];
+          for (const methodParameter of example.methodParameters) {
+            if (
+              methodParameter.exampleValue.schema.type === SchemaType.Constant
+            ) {
+              continue;
             }
-            const methodParameterNames = [];
-            const optionalParams: [string, string][] = [];
-            for (const methodParameter of example.methodParameters) {
+            const parameterName = normalizeName(
+              getLanguageMetadata(methodParameter.exampleValue.language).name,
+              NameType.Parameter,
+              true
+            );
+            const parameterTypeDetails = getTypeForSchema(
+              methodParameter.exampleValue.schema
+            );
+            const parameterTypeName = parameterTypeDetails.typeName;
+            let paramAssignment = "";
+            if (methodParameter.parameter.protocol?.http?.["in"] === "body") {
+              let bodySchemaName = parameterTypeName;
               if (
-                methodParameter.exampleValue.schema.type === SchemaType.Constant
+                methodParameter.exampleValue.schema.type ===
+                  SchemaType.AnyObject ||
+                methodParameter.exampleValue.schema.type === SchemaType.Any
               ) {
-                continue;
-              }
-              const parameterName = normalizeName(
-                getLanguageMetadata(methodParameter.exampleValue.language).name,
-                NameType.Parameter,
-                true
-              );
-              const parameterTypeDetails = getTypeForSchema(
-                methodParameter.exampleValue.schema
-              );
-              const parameterTypeName = parameterTypeDetails.typeName;
-              let paramAssignment = "";
-              if (methodParameter.parameter.protocol?.http?.["in"] === "body") {
-                let bodySchemaName = parameterTypeName;
-                if (
-                  methodParameter.exampleValue.schema.type ===
-                    SchemaType.AnyObject ||
-                  methodParameter.exampleValue.schema.type === SchemaType.Any
-                ) {
-                  bodySchemaName = "Record<string, unknown>";
-                } else if (methodParameter.exampleValue.schema.type !== SchemaType.Dictionary) {
+                bodySchemaName = "Record<string, unknown>";
+              } else if (
+                methodParameter.exampleValue.schema.type !==
+                SchemaType.Dictionary
+              ) {
+                if (parameterTypeName.endsWith("[]")) {
+                  importedTypeSet.add(
+                    parameterTypeName.substring(0, parameterTypeName.length - 2)
+                  );
+                } else {
```

Comment:
This is only part I changed. the rest part is changing because I format this file.